src/Api/User.hs

```diff
@@ -1,46 +1,41 @@
-{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
-{-# LANGUAGE TypeOperators              #-}
+{-# LANGUAGE TypeOperators #-}
 
 module Api.User where
 
-import           Control.Monad.Except
-import           Control.Monad.Reader        (ReaderT, runReaderT)
-import           Control.Monad.Reader.Class
-import           Data.Int                    (Int64)
-import           Database.Persist.Postgresql (Entity (..), fromSqlKey, insert,
-                                              selectFirst, selectList, (==.))
-import           Network.Wai                 (Application)
-import           Servant
-import           Servant.JS                  (vanillaJS, writeJSForAPI)
-
-import           Config                      (App (..), Config (..))
-import           Models
-import           Data.Text (pack)
-
-type UserAPI =
-         "users" :> Get '[JSON] [Entity User]
-    :<|> "users" :> Capture "name" String :> Get '[JSON] (Entity User)
-    :<|> "users" :> ReqBody '[JSON] User :> Post '[JSON] Int64
+import Control.Monad.Except
+import Control.Monad.Reader (ReaderT, runReaderT)
+import Control.Monad.Reader.Class
+import Data.Int (Int64)
+import Database.Persist.Postgresql
+       (Entity(..), (==.), fromSqlKey, insert, selectFirst, selectList)
+import Network.Wai (Application)
+import Servant
+import Servant.JS (vanillaJS, writeJSForAPI)
+
+import Config (App(..), Config(..))
+import Data.Text (Text)
+import Models
+
+type UserAPI
+     = "users" :> Get '[ JSON] [Entity User] :<|> "users" :> Capture "name" Text :> Get '[ JSON] (Entity User) :<|> "users" :> ReqBody '[ JSON] User :> Post '[ JSON] Int64
```

Comment:
Looks like you used `hindent` :) Unfortunately, `hindent` often does undesirable things, so I generally don't like using it to autoformat my project's Haskell files. Please revert the formatting changes.